### PR TITLE
feat: Create card like view for stored credentials

### PR DESCRIPTION
**Pull request by talwinder50**
Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: cmd/wallet-web/src/components/CredentialCards/GeneralCard.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
```
Comment (sudeshrshetty) [inline cmd/wallet-web/src/components/CredentialCards/GeneralCard.vue:7]: is this for credentials which doesn't fall under category?

File: cmd/wallet-web/src/pages/Dashboard.vue [hunk 2]
```diff
@@ -127,6 +194,74 @@ SPDX-License-Identifier: Apache-2.0
         font-size: 6px;
         line-height: 16px;
     }
+    ul.credential-list {
+        padding-left: 0;
+        display: flex;
+        flex-flow: row wrap;
+    }
+
+    li {
```
Comment (soluchok) [inline cmd/wallet-web/src/pages/Dashboard.vue:203]: you are changing `li` which is global, but the `<style>` is not `scoped`.

**Review by sudeshrshetty — COMMENTED** [review]
follow up: 
Can we have just one generic vue card for all the credentials?
we can use vue props to pass icons & credential subject properties.

So that, it can be dynamically updated without code change to add support more VC types.

**Comment by talwinder50** [comment]
Okay i will look into that approach .. do you want me to create todo or do in this pr only ?

**Comment by talwinder50** [comment]
> follow up: 
> 
> Can we have just one generic vue card for all the credentials?
> 
> we can use vue props to pass icons & credential subject properties.
> 
> 
> 
> So that, it can be dynamically updated without code change to add support more VC types.  

Okay i will look into that approach .. do you want me to create todo or do in this pr only

File: cmd/wallet-web/src/pages/Dashboard.vue (hunk 2 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Dashboard.vue:203]: Will make style scoped

**Comment by sudeshrshetty** [comment]
> > follow up:
> > Can we have just one generic vue card for all the credentials?
> > we can use vue props to pass icons & credential subject properties.
> > So that, it can be dynamically updated without code change to add support more VC types.
> 
> Okay i will look into that approach .. do you want me to create todo or do in this pr only

new PR should be fine.

File: cmd/wallet-web/src/components/CredentialCards/GeneralCard.vue (hunk 1 above)
Comment (talwinder50) [reply cmd/wallet-web/src/components/CredentialCards/GeneralCard.vue:7]: yes

**Comment by talwinder50** [comment]
> > > follow up:
> > > Can we have just one generic vue card for all the credentials?
> > > we can use vue props to pass icons & credential subject properties.
> > > So that, it can be dynamically updated without code change to add support more VC types.
> > 
> > 
> > Okay i will look into that approach .. do you want me to create todo or do in this pr only
> 
> new PR should be fine.

created todo #627

File: cmd/wallet-web/src/pages/Dashboard.vue (hunk 2 above)
Comment (talwinder50) [reply cmd/wallet-web/src/pages/Dashboard.vue:203]: done

**Review by soluchok — APPROVED** [review]

**Pull request merged**